CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 ## [unreleased]
 
+### Changed
+
+- animation and shadow effects to be excluded with enact cli `--no-animation` build option
```

Comment:
I think we don't need this change log right now.

Reply:
ok